GNUmakefile

```diff
@@ -48,7 +50,13 @@ attach:
 
 acceptance: fmt
 	export TF_ACC=true && \
-		go test -ldflags="-X '${PKG_VERSION_PATH}.Version=${NEXT_VERSION}-test'" -v -p 1 -parallel 20 -timeout 20m ./pkg/...
+		go test -cover -coverprofile=coverage.txt -ldflags="-X '${PKG_VERSION_PATH}.Version=${NEXT_VERSION}-test'" -v -p 1 -parallel 20 -timeout 20m ./pkg/...
```

Comment:
Why a timeout of 20m?

Reply:
Kinda arbitrary, but if acceptance tests take longer than 20 minutes to finish it is an indication that:
a) Something is wrong with the Artifactory test instance
b) Our tests are too slow
c) The provider is growing 😄 

Any acceptance test suite that takes too long to execute will become less useful. The feedback cycle is too long for us to pay attention to, etc. Ideally we should invest some time and see if we can parallelize more (I did last year for like 30 minutes but didn't make any progress).